pkg/config/config.go

```diff
@@ -219,9 +219,18 @@ func Write() error {
 		return err
 	}
 
+	// Ensure that our configuration directory exists: viper only takes care of
+	// creating the file.
+	configDir := configdir.LocalConfig("grizzly")
+	if _, err := os.Stat(configDir); os.IsNotExist(err) {
+		if err := os.Mkdir(configDir, 0700); err != nil {
```

Comment:
`os.Mkdir` or `os.MkdirAll`?

Reply:
On a linux machine, `configDir` is `~/.config/grizzly`

`os.Mkdir()` lets us create the `grizzly` folder if it doesn't exist while `os.MkdirAll()` would potentially create everything in that path (which practically should never be needed anyway). I decided against using `MkdirAll()` for two reasons: we shouldn't create things we don't own, it would create directories with 0700 permissions (which is probably not what is usually set on `~/.config`)

Reply:
by default ~/.config directory doesn't exists on most Linux distributions by default.

Without that, you'll run into:
```
$ grr config create-context grafana-onprem
FATAL mkdir /home/<user>/.config/grizzly: no such file or directory
```

I think `os.MkdirAll()` is better for this. Or, change the requirement for the config to use a multiple directories path.. so instead of `~/.config/grizzly` maybe `~/grizzly` is enough.